### DISCUSSION TITLE
Metrics for CAM7 and CESM3 Runs

### DISCUSSION THREAD

**Discussion by swrneale** [NCAR/amwg_dev#529] [Discussion about coupled runs]
What are are our target metrics for CAM7 and CESM3 simulated climate.

**Comment by PeterHjortLauritzen** [comment]
Here is my list from CAM7 meeting to kick-off the discussion: (these are subjective metrics though; and not prioritized)

- El Niños are not transitioning to La Nina’s as in observations and CESM2. Not significantly worse than CESM2 metric?

- Reduce "dry bias" in standard deviation of precip in East Pacific?

- Acceptable MJO

- “Less Labrador Sea freezes”

- Acceptable water vapor tape recorder

- Acceptable QBO

- Acceptable polar night jets (seasonal cycle of polar vortex)

- PSL, U200

- Acceptable SWCF and LWCF? 

- Taylor diagram?

What else? How do we define acceptable?

See also CAM7 meeting slides for 5/3/2024:

https://docs.google.com/presentation/d/1eg5vO6CPrW8SU2qAKi3I15_b5PRN4INFEDK5uBFBRig/edit#slide=id.g26fe79b34f7_1_280

**Reply by swrneale** [reply]
I think a starting point would be to assess these biases in CAM6 and CESM2, and then decide what tolerance we have for these biases in CAM7/CESM3.  Low ball would be that more of these metric biases are better in CESM3/CAM7 than in CESM2/CAM6. High ball would be that they are all better :)
